app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/PaginationConstants.kt

```diff
@@ -0,0 +1,4 @@
+package com.architectcoders.arquitectomarvel.ui.main.pagination
+
+const val PAGE_SIZE = 18
```

Comment:
@joseantoniocampillo these constants should go to the Constant.kt file

Reply:
Already created a package for pagination it seemed good to me to place to host this constants.

Reply:
IMO, all constants should go to the same place to be located easily.